src/gt4py/next/program_processors/runners/dace_fieldview/gtir_to_tasklet.py

```diff
@@ -59,6 +59,8 @@ class SymbolExpr:
     dace.nodes.Node,
     Optional[str],
 ]
+# special connection from map entry node to a tasklet node without arguments
+TaskletConnection: TypeAlias = dace.nodes.Tasklet
```

Comment:
Maybe you missed it, but `InputConnection` is a tuple, not a union. So the following point is wrong:

> The definition of TaskletConnection is redundant, since it is included in InputConnection

Both `InputConnection` and `TaskletConnection` are lowered to edges on the map entry node:

- `InputConnection` is a tuple containing the elements needed to "draw" an edge with a data memlet.
- `TaskletConnection` is used to "draw" an empty edge to tasklets that do not have inputs.

Reply:
Addressed by refactoring-PR #1685